frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -864,6 +883,11 @@ const QueueItem = ({
                       <b> {` ${patientFullName} `} </b> has not been completed.
                       Do you want to submit results anyway?
                     </p>
+                  ) : isCorrection === true ? (
+                    <p>
+                      TODO: Rebecca to provide alternate copy for canceling a
```

Comment:
@Rebecca-LM _cough, cough_ 😉 

Currently, the message displayed to users when cancelling a test is:

<img width="512" alt="Screen Shot 2022-04-08 at 3 59 48 PM" src="https://user-images.githubusercontent.com/27730981/162517997-7348a6bc-6e2f-469e-9dc2-67226f4416f9.png">

@vz3 suggested that we tweak this in cases where a user is attempting to cancel a _correction_ rather than a normal test, just for extra clarity.

Reply:
Got it! 🖖 

<img width="792" alt="Screen Shot 2022-04-08 at 5 20 40 PM" src="https://user-images.githubusercontent.com/27730981/162533836-e707dafb-38bd-4166-b935-ad1ad98049fb.png">